src/dealDashboard/dealInfo/dealInfo.scss

```
@@ -153,8 +153,20 @@ deal-info {
         .representative-dao {
           font-weight: 700;
           flex: 1;
+          a {
+            color: $Neutral02;
+          }
         }
       }
     }
   }
+
+  .percent{
```

```suggestion
  .reward-percent{
```

a bit more descriptive would be nice